python/ql/test/experimental/dataflow/regression/pointsto.ql

```diff
@@ -0,0 +1,28 @@
+private import python
+import experimental.dataflow.DataFlow
+
+predicate pointsToOrigin(DataFlow::DataFlowCfgNode pointer, DataFlow::DataFlowCfgNode pointed) {
```

Comment:
Could we rename `pointed` to `origin`? To me they look too similar (and `pointed` is too "vague").
Alternatively, `nodeFrom` and `nodeTo`.

Reply:
I like `origin`, will do :-)